fr.etnadji.dispremi.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "fr.etnadji.dispremi",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "dispremi",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=dri",
+        "--filesystem=~/.config/dispremi.json",
```

Comment:
You should use xdg-data-dirs instead of hardcoding the path to .config

Reply:
Done.